.github/workflows/black.yml

```diff
@@ -64,13 +64,13 @@ jobs:
             new_ver = ver + 0.001;
             printf("    script_version = \"%.3f\"\n", new_ver);
           }
-          !/script_version = "/ { print $0 }' coreutils.py > coreutils.py.tmp && mv coreutils.py.tmp coreutils.py
+          !/script_version = "/ { print $0 }' wemod_launcher/coreutils.py > wemod_launcher/coreutils.py.tmp && mv wemod_launcher/oreutils.py.tmp wemod_launcher/coreutils.py
```

Comment:
Found a typo here you have wemod_launcher/oreutils.py.tmp but its wemod_launcher/coreutils.py.tmp
But i think you changed it to .core_utils.py or something like it.

Reply:
Resolved.